Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -2306,6 +2367,77 @@ void qMRMLSliceControllerWidget::showReformatWidget(bool show)
     }
 }
 
+//---------------------------------------------------------------------------
+void qMRMLSliceControllerWidget::showReconstructionWidget(bool show)
+{
+  Q_D(qMRMLSliceControllerWidget);
+
+  vtkSmartPointer<vtkCollection> nodes = d->saveNodesForUndo("vtkMRMLSliceNode");
+  if (!nodes.GetPointer())
+    {
+    return;
+    }
+  vtkMRMLSliceNode* node = nullptr;
+  vtkMRMLSliceLogic* sliceLogic = nullptr;
+  // vtkCollectionSimpleIterator it;
+  for (int i = 0; i < nodes->GetNumberOfItems(); i++)
+  {
+    node = static_cast<vtkMRMLSliceNode*>(nodes->GetItemAsObject(i));
+    sliceLogic = static_cast<vtkMRMLSliceLogic*>(d->SliceLogics->GetItemAsObject(i));
+    if (node == this->mrmlSliceNode() || this->isLinked())
+      {
+        node->SetSlabReconstructionVisibility(show);
+        auto resliceBG = sliceLogic->GetBackgroundLayer()->GetReslice();
+        auto resliceFG = sliceLogic->GetForegroundLayer()->GetReslice();
+        if (show)
+        {
+          auto slabType = node->GetSlabReconstructionType();
+          if (slabType == vtkMRMLSliceNode::SliceReconstructionMax)
+          {
+            resliceBG->SetSlabModeToMax();
+            resliceFG->SetSlabModeToMax();
+          }
+          else if (slabType == vtkMRMLSliceNode::SliceReconstructionMin)
+          {
+            resliceBG->SetSlabModeToMin();
+            resliceFG->SetSlabModeToMin();
+          }
+          else if (slabType == vtkMRMLSliceNode::SliceReconstructionMean)
+          {
+            resliceBG->SetSlabModeToMean();
+            resliceFG->SetSlabModeToMean();
+          }
+          else if (slabType == vtkMRMLSliceNode::SliceReconstructionSum)
+          {
+            resliceBG->SetSlabModeToSum();
+            resliceFG->SetSlabModeToSum();
+          }
+          auto volumeBG = sliceLogic->GetLayerVolumeNode(vtkMRMLSliceLogic::LayerBackground);
+          if (volumeBG)
+          {
+            auto maxVolumeSpacingBG = volumeBG->GetMaxSpacing();
+            resliceBG->SetSlabNumberOfSlices(int(node->GetSlabThickness() / (maxVolumeSpacingBG ? maxVolumeSpacingBG : 1)));
+          }
+
+          auto volumeFG = sliceLogic->GetLayerVolumeNode(vtkMRMLSliceLogic::LayerForeground);
+          if (volumeFG)
+          {
+            auto maxVolumeSpacingFG = volumeFG->GetMaxSpacing();
+            resliceFG->SetSlabNumberOfSlices(int(node->GetSlabThickness() / (maxVolumeSpacingFG ? maxVolumeSpacingFG : 1)));
+          }
+        }
+        else
+        {
+          resliceBG->SetSlabNumberOfSlices(1);
+          resliceFG->SetSlabNumberOfSlices(1);
+        }
+        resliceBG->SetSlabSliceSpacingFraction(0.3);
```

Comment:
Add a comment describing why it is `0.3` and you could add a internal variable with the hard-coded value.